README.md

```diff
@@ -118,6 +118,11 @@ Other languages are periodically pulled from [Transifex](https://www.transifex.c
 
 ## Releasing
 
+- Manually test a few things that don't transfer well to automated testing:
```

Comment:
I would move this to a section called Pre-release testing? Otherwise it gets too lengthy

Reply:
@rafaelramalho19 - yep, planning to rewrite all this stuff under https://github.com/ipfs-shipyard/ipfs-desktop/issues/1670 so will leave as is for now if you don't mind. 😊

Reply:
Alright 👍